assets/js/partners/contact.js

```diff
@@ -3,6 +3,7 @@ const BLOCKED_DOMAINS = [
   'hotmail.com',
   'icloud.com',
   'live.com',
+  'me.com',
```

Comment:
this seems like a good candidate for a config that we maybe add as a `data-blocked-emails` attribute and then update it in the Cloud.gov Pages UI?